src/main/java/bio/terra/cli/command/shared/options/ResourceUpdate.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.cli.serialization.userfacing.inputs.UpdateResourceParams;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the relevant options for updating a Terra resource: {@link
+ * ResourceName}, --new-name, and {@link ResourceDescription}.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class ResourceUpdate {
+  @CommandLine.Mixin public ResourceName resourceNameOption;
+
+  @CommandLine.Option(
+      names = "--new-name",
```

Comment:
consider changing workspace update to also take `--new-name` to match this for user consistency.

Reply:
I think these cases are a little different because `name` is a unique identifier for resources, but not for workspaces. Similar to how both use `--description` instead of `--new-description`.